components/developer/gdb/gdb.p5m

```diff
@@ -34,7 +37,7 @@ set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
-file path=usr/bin/gdb
+file path=usr/bin/gdb pkg.depend.bypass-generate=.*/libpython3.9.so.*
```

Comment:
I'd like to see more details why is this needed.  If it was able to automatically resolve dependency on `libpython3.7m.so.1.0` then it should also work for `libpython3.9.so.1.0` which lives in the same directory.

Reply:
Added some details. Unfortunately I lack the time and experience to hunt down the bug inside pkg but it is definetly a bug that I needed to workaround.